specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -104,30 +102,30 @@ requirements listed in the Servlet specification.
 The fourth category is the most complex. As a
 first example, since a Web Profile product is required to implement the
 Servlet technology, it must also follow all general requirements for
-Java EE web containers in the Platform specification. Additionally, it
+Jakarta EE web containers in the Platform specification. Additionally, it
 must follow all security requirements in the Platform specification that
-pertain to Java EE web containers, all interoperability requirements for
+pertain to Jakarta EE web containers, all interoperability requirements for
 such containers, etc. Furthermore, since a Web Profile product must
-implement the Java Transaction API (JTA), it must also satisfy all the
+implement the Jakarta Transaction API (JTA), it must also satisfy all the
```

Comment:
Jakarta Transactions

Reply:
And to be clear, it should be "... implement Jakarta Transactions, ..." or
"... implement the Jakarta Transactions specification, ..."